Assets/Scripts/Threats/FixedWingThreat.cs

```diff
@@ -17,5 +24,120 @@ protected override void UpdateReady(double deltaTime) {}
+      // Combine the accelerations
+      accelerationInput = pnAcceleration + speedAdjustmentAcceleration;
+
+      // Clamp the total acceleration
+      float maxAcceleration = CalculateMaxAcceleration();
+      if (accelerationInput.magnitude > maxAcceleration) {
+        accelerationInput = accelerationInput.normalized * maxAcceleration;
+      }
+    }
+
+    // Calculate and set the total acceleration
+    Vector3 acceleration = CalculateAcceleration(accelerationInput, compensateForGravity: true);
+    GetComponent<Rigidbody>().AddForce(acceleration, ForceMode.Acceleration);
+  }
+  private void UpdateWaypointAndPower() {
+    // Get the next waypoint and power setting from the attack behavior
+    // TODO: Implement support for SENSORS to update the track on the target position
+    (_currentWaypoint, _currentPowerSetting) = _attackBehavior.GetNextWaypoint(transform.position, _target.transform.position);
+  }
+
+  private Vector3 CalculateAccelerationCommand(SensorOutput sensorOutput) {
+    // Implement Proportional Navigation guidance law
+    Vector3 accelerationCommand = Vector3.zero;
+
+    // Extract relevant information from sensor output
+    float los_rate_az = sensorOutput.velocity.azimuth;
+    float los_rate_el = sensorOutput.velocity.elevation;
+    float closing_velocity =
+        -sensorOutput.velocity
+             .range;  // Negative because closing velocity is opposite to range rate
+
+    // Navigation gain (adjust as needed)
+    float N = _navigationGain;
+
+    // Calculate acceleration commands in azimuth and elevation planes
+    float acc_az = N * closing_velocity * los_rate_az;
+    float acc_el = N * closing_velocity * los_rate_el;
+
+    // Convert acceleration commands to craft body frame
+    accelerationCommand = transform.right * acc_az + transform.up * acc_el;
+
+    // Clamp the acceleration command to the maximum acceleration
+    float maxAcceleration = CalculateMaxAcceleration();
+    accelerationCommand = Vector3.ClampMagnitude(accelerationCommand, maxAcceleration);
+
+    // Update the stored acceleration command for debugging
+    _accelerationCommand = accelerationCommand;
+    return accelerationCommand;
+  }
+
+  private Vector3 CalculateSpeedAdjustmentAcceleration() {
+    // Get the target velocity for the current power setting
+    float targetSpeed = PowerTableLookup(_currentPowerSetting);
+
+    // Calculate the current speed
+    float currentSpeed = GetVelocity().magnitude;
+
+    // Speed error
+    float speedError = targetSpeed - currentSpeed;
+
+    // Proportional gain for speed control
+    float speedControlGain = 10.0f; // Adjust this gain as necessary
+
+    // Desired acceleration to adjust speed
+    float desiredAccelerationMagnitude = speedControlGain * speedError;
+
+    // Limit the desired acceleration
+    float maxAcceleration = CalculateMaxAcceleration();
+    desiredAccelerationMagnitude = Mathf.Clamp(desiredAccelerationMagnitude, -maxAcceleration, maxAcceleration);
```

Comment:
hm total acceleration is sqrt(forward^2 + normal^2).  is it worth checking both?

basically it'll turn into some vector whose direction is a weighted combination of the normal acceleration and the forward acceleration, and this total vector will be scaled to max acceleration